src/protocol_pro_2.cpp

```diff
@@ -41,7 +42,53 @@ Pro2ProtocolObject::Pro2ProtocolObject(
       std::thread([this]() { this->motors_control_loop(30); });
 }
 
-void Pro2ProtocolObject::update_drivetrim(double value) { trimvalue_ += value; }
+void Pro2ProtocolObject::get_params_file() {
+  // open yaml
+  std::ifstream robotconfig;
+  std::string const HOME = std::getenv("HOME") ? std::getenv("HOME") : ".";
+  robotconfig.open(HOME + "/rover.config");
+  if (robotconfig.is_open()) {
+    std::string line;
+    std::cerr << "Loading config from " + HOME + "/rover.config" << std::endl;
+    while (std::getline(robotconfig, line)) {
+      std::vector<std::string> key =
+          split(line, ":");  // get key-value pair into vector
+      std::vector<std::string> values =
+          split(key[1], " ");  // get all the values from this key
+      if (key[0] == "trim") {  // check for key
```

Comment:
please enumerate the keys in a header file (even if there is only 1 key right now).